src/components/table/style.css

```
@@ -1,9 +1,14 @@
 :host {
 	display: table;
 	text-align: left;
-	border-collapse: collapse;
-	border: 3px solid rgb(var(--smoothly-dark-color), 0.5);
-	width: 100%;
+	border: 1px solid rgb(var(--smoothly-dark-color));
+	width: 80%;
 	box-sizing: border-box;
 	background-color: rgb(var(--smoothly-default-color));
+	margin: 1em 9.5%  ;
```

Please fix the prettier style lint in a PR before this so you can fun `npm run fix` and get it sorted automatically and preferably even get it to lint on save.
```suggestion
	margin: 1em 9.5%;
```

---

created a separate branch, lint-fix, and fixed the  prettier-stylelint issue. then created a PR with that branch, i then merged lint-fix into this branch